src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -77,6 +77,8 @@
     </EnumProperty>
     <StringListProperty Name="AvailablePlatforms" Separator="," />
     <BoolProperty Name="Optimize" Description="Should compiler optimize output?" />
+    <!-- Same as MSBuildProjectDirectory but has a directory separator '\' appended at the end.-->
+    <StringProperty Name="ProjectDir" Visible="false"/>
```

Comment:
$ProjectDir is a built-in property. and [ProjectDir](https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.Common.CurrentVersion.targets#L285) is nothing but MSBuildProjectDirectory appended with a slash